<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="detalle_cto_certificadosV2-20241107165046" tests="1" file=".py" time="10.351" timestamp="2024-11-07T16:50:56" failures="0" errors="1" skipped="0">
	<testcase classname="detalle_cto_certificadosV2" name="test_detail_certificates" time="10.351" timestamp="2024-11-07T16:50:56" file="CtaCtoDtalCertificado2 copy.py" line="25">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=130.0.6723.116)
Stacktrace:
	GetHandleVerifier [0x00007FF609CA38A5+3004357]
	(No symbol) [0x00007FF609939970]
	(No symbol) [0x00007FF6097E582A]
	(No symbol) [0x00007FF6097BFA75]
	(No symbol) [0x00007FF609862C77]
	(No symbol) [0x00007FF60987B351]
	(No symbol) [0x00007FF60985B983]
	(No symbol) [0x00007FF609827628]
	(No symbol) [0x00007FF609828791]
	GetHandleVerifier [0x00007FF609CCA00D+3161901]
	GetHandleVerifier [0x00007FF609D1E060+3506048]
	GetHandleVerifier [0x00007FF609D1400D+3465005]
	GetHandleVerifier [0x00007FF609A90EEB+830987]
	(No symbol) [0x00007FF60994467F]
	(No symbol) [0x00007FF6099409D4]
	(No symbol) [0x00007FF609940B6D]
	(No symbol) [0x00007FF609930149]
	BaseThreadInitThunk [0x00007FFF3E62257D+29]
	RtlUserThreadStart [0x00007FFF4022AF08+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\CtaCtoDtalCertificado2 copy.py", line 28, in test_detail_certificates
    self.login_helper.select_tenant()
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\loginhelper.py", line 18, in select_tenant
    select_tenant = find_element_by_xpath(self.driver, "/html/body/app-root/app-layout/app-vertical/div/div/div/div/app-home-main/div/div[1]/app-tenant-main/app-tenant[8]/div/div/img")
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\findElements.py", line 14, in find_element_by_xpath
    return WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, xpath)))
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 84, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 748, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=130.0.6723.116)
Stacktrace:
	GetHandleVerifier [0x00007FF609CA38A5+3004357]
	(No symbol) [0x00007FF609939970]
	(No symbol) [0x00007FF6097E582A]
	(No symbol) [0x00007FF6097BFA75]
	(No symbol) [0x00007FF609862C77]
	(No symbol) [0x00007FF60987B351]
	(No symbol) [0x00007FF60985B983]
	(No symbol) [0x00007FF609827628]
	(No symbol) [0x00007FF609828791]
	GetHandleVerifier [0x00007FF609CCA00D+3161901]
	GetHandleVerifier [0x00007FF609D1E060+3506048]
	GetHandleVerifier [0x00007FF609D1400D+3465005]
	GetHandleVerifier [0x00007FF609A90EEB+830987]
	(No symbol) [0x00007FF60994467F]
	(No symbol) [0x00007FF6099409D4]
	(No symbol) [0x00007FF609940B6D]
	(No symbol) [0x00007FF609930149]
	BaseThreadInitThunk [0x00007FFF3E62257D+29]
	RtlUserThreadStart [0x00007FFF4022AF08+40]

]]></error>
	</testcase>
</testsuite>
